modules/failure-handler/src/main/java/org/apache/ignite/internal/failure/FailureManager.java

```diff
@@ -168,6 +196,24 @@ private void initFailureHandler() {
             return;
         }
 
+        dumpThreadsOnFailure = configuration.dumpThreadsOnFailure().value();
```

Comment:
Would it make sense to make these properties dynamic (that is, listen to their changes)?

Reply:
There is a separate ticket to address that https://issues.apache.org/jira/browse/IGNITE-21461